diff --git a/node_modules/subscriptions-transport-ws/dist/client.js b/node_modules/subscriptions-transport-ws/dist/client.js
index 3202dcd..ec570ce 100644
--- a/node_modules/subscriptions-transport-ws/dist/client.js
+++ b/node_modules/subscriptions-transport-ws/dist/client.js
@@ -122,11 +122,12 @@ var SubscriptionClient = (function () {
                 this.unsubscribeAll();
                 this.sendMessage(undefined, message_types_1.default.GQL_CONNECTION_TERMINATE, null);
             }
-            this.client.close();
-            this.client.onopen = null;
-            this.client.onclose = null;
-            this.client.onerror = null;
-            this.client.onmessage = null;
+            const client = this.client;
+            client.close();
+            client.onopen = null;
+            client.onclose = null;
+            client.onerror = null;
+            client.onmessage = null;
             this.client = null;
             this.eventEmitter.emit('disconnected');
             if (!isForced) {
